pkgs/top-level/rust-packages.nix

```diff
@@ -8,14 +8,14 @@
 
 let
   version = "2016-02-02";
```

Comment:
Please also update the date, so that we can tell how long ago was the last update. Thanks!